x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,278 @@
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatDescription"
+            defaultMessage="Follow {link}."
+            values={{
+              link: (
+                <EuiLink
+                  href={`${ELASTIC_WEBSITE_URL}guide/en/beats/metricbeat/${DOC_LINK_VERSION}/metricbeat-installation.html`}
+                  target="_blank"
+                >
+                  <FormattedMessage
+                    id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatLinkText"
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Elasticsearch x-pack module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+            metricbeat modules enable elasticsearch-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Elasticsearch monitoring metrics from {url}.
+              If the local Elasticsearch server has a different address,
+              you must specify it via the hosts setting in the {module} file."
+              values={{
+                module: (
+                  <Monospace>modules.d/elasticsearch-xpack.yml</Monospace>
+                ),
+                url: (
+                  <Monospace>http://localhost:9200</Monospace>
+                )
+              }}
+            />
+          </p>
+        </EuiText>
+        {securitySetup}
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.configureMetricbeatTitle', {
+      defaultMessage: 'Configure Metricbeat to send to the monitoring cluster'
+    }),
+    children: (
+      <Fragment>
+        <EuiText>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.configureMetricbeatDescription"
+            defaultMessage="Make these changes in your {file}."
+            values={{
+              file: (
+                <Monospace>metricbeat.yml</Monospace>
+              )
+            }}
+          />
+        </EuiText>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`
```

Comment:
Watch out for these extra line breaks, they cause empty lines in the code blocks:

<img src="https://d.pr/free/i/IMGexw+" width="50%" />

Reply:
Nice catch!

<img width="885" alt="Screen Shot 2019-06-14 at 10 03 07 AM" src="https://user-images.githubusercontent.com/56682/59514886-d5cb9c80-8e8b-11e9-8db4-7f8a84dd70a3.png">